src/pages/Homepage/Homepage.vue

```diff
@@ -4,6 +4,11 @@
 		:footer-theme="footerTheme"
 	>
 		<component :is="activeHomepage" />
+		<router-link
```

Comment:
Just a temporary link to test prefetching from the homepage to page-two

Reply:
I think you can just use the link from `ui-site-map` right?